compiler/src/iree/compiler/Codegen/LLVMCPU/Passes.cpp

```diff
@@ -56,6 +56,18 @@ static llvm::cl::opt<bool> clEnableMicrokernels(
 // pipeline.
 extern llvm::cl::opt<std::string> clCPUCodegenTransformDialectFileName;
 
+//===---------------------------------------------------------------------===//
+// Default Linalg code generation options for CPU backend
+//===---------------------------------------------------------------------===//
+
+struct LinalgCPUVectorLoweringPassOptions : LinalgVectorLoweringPassOptions {
```

Comment:
Might be a nit, but doing this in the constructor makes it harder to read for me (cause some options are set in the constructor, and some in the pass pipeline. I am fine with it if you insist, but I'd rather make it explicit in each pipeline so its clear whats happening (at the cost of code duplication)

Reply:
My main goal here is to define the default set of options for all the CPU pipelines and refactor them to a single place to make it less error prone (i.e., if we forget to forget to set one option in one of the specific pipelines below). The idea was to keep in the pipelines those flags that are different from pipeline to pipeline. Note that `LinalgVectorLoweringPassOptions` is already setting up some defaults that not ideal for CPU so this class is just overwriting those. I can't think of a better way to do this refactoring. Any suggestion?